### PR TITLE
chore: Update all references of 'Twitter' to 'X.com'

### PR DESCRIPTION
**Pull request by NikolaiKryshnev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Before submitting an enhancement suggestion, check if there is already a filed o
 First of all, you can validate it with the community, our official channels are:
 
 - [Discord](https://discord.gg/hemixyz)
-- [Twitter](https://twitter.com/hemi_xyz)
+- [Twitter](https://x.com/hemi_xyz)
```
Comment (gabmontes) [inline CONTRIBUTING.md:87]: The name should be changed too.

```suggestion
- [X](https://x.com/hemi_xyz)
```

**Review by gabmontes — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (NikolaiKryshnev) [reply CONTRIBUTING.md]: Changes applied as per your suggestion. Please review.

Comment (NikolaiKryshnev) [reply CONTRIBUTING.md]: @gabmontes Hi! Just wanted to kindly follow up on this PR. Please let me know if there's anything else needed from my side to proceed with the merge. 😊

**Review by gndelia — APPROVED** [review]

**Review by ArturDolzan — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]
✨

**Pull request merged**